rest/perf_routes.go

```diff
@@ -283,3 +283,41 @@ func (h *perfGetChildrenHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 	return gimlet.NewJSONResponse(perfResults)
 }
+
+///////////////////////////////////////////////////////////////////////////////
+//
+// POST /perf/signal_processing/recalculate
+
+type perfSignalProcessingRecalculateHandler struct {
+	sc data.Connector
+}
+
+func makePerfSignalProcessingRecalculate(sc data.Connector) gimlet.RouteHandler {
+	return &perfSignalProcessingRecalculateHandler{
+		sc: sc,
+	}
+}
+
+func (h *perfSignalProcessingRecalculateHandler) Factory() gimlet.RouteHandler {
+	return &perfSignalProcessingRecalculateHandler{
+		sc: h.sc,
+	}
+}
+
+func (h *perfSignalProcessingRecalculateHandler) Parse(_ context.Context, r *http.Request) error {
+	var err error
+	return errors.Wrap(err, "failed to parse request")
```

Comment:
this always returns nil,.

Reply:
What am I supposed to return? I don't really need to parse anything, was going by whats on the other routes.

Reply:
`return nil` is acceptable, the code above is confusing because it looks like it's returning an error, rather than nil, but you have to know about the behavior of errors.Wrap() to know what's happening here.

I think the goal should be "be as clear as possible,"